Jenkinsfile

```diff
@@ -0,0 +1,83 @@
+// This Jenkinsfile is used by Jenkins to run the 'SearchIndexer' step of Reactome's release.
+// It requires that the 'DiagramConverter' step has been run successfully before it can be run.
+
+import org.reactome.release.jenkins.utilities.Utilities
+
+// Shared library maintained at 'release-jenkins-utils' repository.
+def utils = new Utilities()
+
+pipeline{
+	agent any
+
+	stages{
+		// This stage checks that upstream project 'DiagramConverter' was run successfully.
+		stage('Check DiagramConverter build succeeded'){
+			steps{
+				script{
+                    			utils.checkUpstreamBuildsSucceeded("File-Generation/job/DiagramConverter/")
+				}
+			}
+		}
+		// This stage builds the jar file using maven.
+		stage('Setup: Build jar file'){
+			steps{
+				script{
+					sh "mvn clean package"
+				}
+			}
+		}
+		// Execute the jar file, producing data-export files. This step uses a bash script, 'run-indexer.sh' that is found in the 'scripts' folder.
+		// This step requires both the neo4j and solr credentials.
+		stage('Main: Run Search-Indexer'){
+			steps{
+				script{
+					withCredentials([usernamePassword(credentialsId: 'neo4jUsernamePassword', passwordVariable: 'neo4jPass', usernameVariable: 'neo4jUser')]){
+						withCredentials([usernamePassword(credentialsId: 'solrUsernamePassword', passwordVariable: 'solrPass', usernameVariable: 'solrUser')]){
+							sh "./scripts/run-indexer.sh solrpass=${solrPass} neo4jpass=${neo4jPass} iconsdir=${env.ICONS_ABS_PATH} ehlddir=${env.ABS_DOWNLOAD_PATH}/${currentRelease}/ehld/ maildest=${env.RELEASE_DEVELOPER_EMAIL}"
```

Comment:
Is it necessary to use `run-indexer.sh`?

Reply:
Its not, maybe I will look into running it without this script during the dry release run

Reply:
We can add another optional param to the run-indexer.sh `jenkins` and in line 36 we'll pass `jenkins=true`, with that we'll be able to bypass the branch changes.

Reply:
This would work too, but I think the safest approach (and the one I'd like to implement) would be my suggestion [here](https://github.com/reactome/search-indexer/pull/5#discussion_r568003222)